package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "type": "module",
```

Comment:
As a follow-up task, I think we should consider pnpm in light of the recent npm attack.